tests.py

```diff
@@ -748,6 +748,44 @@ def test_get_fully_coherent_BSGL(self):
         )
         self.assertTrue(lnBSGL > 0)
 
+    def test_cumulative_twoF(self):
+
+        search = pyfstat.ComputeFstat(
+            tref=self.Writer.tref,
+            sftfilepattern=self.Writer.sftfilepath,
+            search_ranges=self.search_ranges,
+        )
+        taus, twoF_cumulative = search.calculate_twoF_cumulative(
+            self.Writer.F0,
+            self.Writer.F1,
+            self.Writer.F2,
+            self.Writer.Alpha,
+            self.Writer.Delta,
+            tstart=self.Writer.tstart,
+            tend=self.Writer.tend(),
+        )
+        twoF_cumulative = twoF_cumulative[-1]
```

Comment:
Should we also test that the `[0]` element is close to 0 and the middle element close to `twoF/2`? Otherwise we're not testing against another regression of the "always takes full data stretch" type. Might require too much tolerancde fiddling, though.

Reply:
Maybe we can compare ranges on top of it?

Like `twoF_cumulative[-1] - twoF_cumulative[0]` and compare that to `twoF` on the whole stream minus `twoF` on the first two ours?